src/style.css

```diff
@@ -1 +1,46 @@
-/* add styles here */
+* {
```

Comment:
Using the universal selector `*` for resetting styles can impact performance. Consider using more specific selectors.